ci/pipeline.yml

```diff
@@ -3,14 +3,14 @@ jobs:
 - name: deploy-development-bosh
   serial: true
   plan:
-  - aggregate:
+  - in_parallel:
     - get: ca-cert-store
     - get: bosh-deployment
       trigger: true
     - get: bosh-config
       resource: bosh-config-development
       trigger: true
-    - get: bosh-stemcell
+    - get: bosh-stemcell-xenial
```

Comment:
What does switching from `bosh-stemcell` to `bosh-stemcell-xenial` do?

Reply:
The default resource points to trusty, this makes it clear we leverage xenial stemcells.